cmd/server/main.go

```diff
@@ -388,6 +389,15 @@ func allowPublicInitiativeLookups(r *http.Request) bool {
 	return allowPublicInitiativeLookupsRegexp.MatchString(r.URL.Path)
 }
 
+var allowPublicAnalysisDownloadsRegexp = regexp.MustCompile(`^/report/.*$`)
```

Comment:
lol sorry to keep beating the anti-regexp drum here, but if we want to say "any path starting with `/report/`", that's just `strings.HasPrefix(r.URL.Path, "/report/")`

Reply:
Lol, that one is fair :laughing: